src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
Consider making 'fruitStorage' private and providing public static methods to interact with it. Direct access to the storage map breaks encapsulation and can lead to issues with data integrity if it is modified from outside the class without any control.